lib/axon/layers.ex

```diff
@@ -1407,11 +1407,11 @@ defmodule Axon.Layers do
   ## Options
 
     * `:rate` - dropout rate. Used to determine probability a connection
-      will be dropped. Required.
+    will be dropped. Required.
```

Comment:
We usually keep lists indented like that, this is not an issue :) Please correct other places

```suggestion
      will be dropped. Required.
```